src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperBlockBuilder.cs

```diff
@@ -75,6 +83,12 @@ public TagHelperBlockBuilder(TagHelperBlock original)
         }
 
         /// <summary>
+        /// Text used as a required prefix when matching HTML start and end tags in the Razor source to available 
+        /// tag helpers.
+        /// </summary>
+        public string Prefix { get; set; }
```

Comment:
sounds right but please clarify "calculate it": what needs to be calculated?  since a `TagHelperBlockBuilder` is created when full name (from the Razor source) is known, I hope you don't mean `FullTagName`.